pyprep/prep_pipeline.py

```diff
@@ -54,16 +54,38 @@ class PrepPipeline:
     def __init__(self, raw, prep_params, montage, ransac=True, random_state=None):
         """Initialize PREP class."""
         self.raw = raw.copy()
-        self.ch_names = self.raw.ch_names
+
+        # separe eeg and non eeg channels
+        self.raw_non_eeg = self.raw.copy()
+        self.chs_all = self.raw.ch_names.copy()
+        self.chs_types = self.raw.get_channel_types()
+        self.chs_eeg = [
+            self.chs_all[i]
+            for i in range(len(self.chs_all))
+            if self.chs_types[i] == "eeg"
+        ]
+        self.chs_non_eeg = set(self.chs_all) - set(self.chs_eeg)
+        self.raw.pick_channels(self.chs_eeg)
+        self.raw_non_eeg.pick_channels(self.chs_non_eeg)
+
         self.raw.set_montage(montage)
-        self.raw.pick_types(eeg=True, eog=False, meg=False)
-        self.ch_names_eeg = self.raw.ch_names
+
         self.EEG_raw = self.raw.get_data() * 1e6
         self.prep_params = prep_params
+        if self.prep_params["ref_chs"] == "eeg":
+            self.prep_params["ref_chs"] = self.chs_eeg
+        if self.prep_params["reref_chs"] == "eeg":
+            self.prep_params["reref_chs"] = self.chs_eeg
```

Comment:
This needs to be updated in the docstring above, and I suggest to make this the new default.

```Python
        - ref_chs : list | 'eeg'
            - A list of channel names to be used for rereferencing.
               Can be a str 'eeg' to use all EEG channels (default).
        - reref_chs : list | 'eeg'
            - A list of channel names to be used for line-noise removed, and
              referenced. Can be a str 'eeg' to use all EEG channels (default).
```

Reply:
I wanted to make it a default but we have the following problem, the input dictionary defines ref, reref and the line freqs. The ref and reref as we know are usually 'eeg' channels (unless is a really special case); but the line_freqs can be 50 or 60.., Which line_freq should we assume ? Would a default line_freq make errors more common as users can init the prep with no params?

Reply:
I will add the doc string without the default for now until we clarify this.

Also should we consider to pass them as normal function arguments rather than a dictionary to avoid the mentioned problem? We could even support both dictionary and argument inputs but maybe it would a little bit confusing. 

What do you think?